sdk/ml/azure-ai-ml/azure/ai/ml/entities/_job/sweep/sampling_algorithm.py

```diff
@@ -15,7 +15,13 @@
 
 
 class SamplingAlgorithm(ABC, RestTranslatableMixin):
+    """Base class for sampling algorithms.
+
+    You should not instantiate this class directly. Instead, use of its subclasses.
```

Comment:
nit: Missing word "one":
```suggestion
    You should not instantiate this class directly. Instead, use one of its subclasses.
```